client/components/AllProducts.js

```diff
@@ -16,24 +16,39 @@ class AllProducts extends Component {
   }
 
   addProductToCart(product) {
-    console.log('product to be added: ', product);
+    console.log("product to be added: ", product);
     this.props.updateCart(this.props.user, product, 1);
   }
 
   render() {
-    console.log('these are my props: ', this.props);
+    console.log("these are my props: ", this.props);
     return (
-      <div>
-        <h1>This is the AllProducts component</h1>
+      <div id="all-products">
+        <video
```

Comment:
@Nievescs20 checkout browser console warnings on this component for "Invalid DOM property". It seems to want all these video properties in camelCase, e.g. playsInLine, controlsList, autoPlay etc

Reply:
not sure about these warnings. if i change property to camelCase the video element breaks